uiplib/scrape.py

```diff
@@ -37,17 +37,38 @@ def get_images(url, directory, count):
     '''
     image_links = []
     no_of_images = int(count)
-    page = make_json(url)
-    for sub in page['data']['children']: # structure of reddit API
-        for image in sub['data']['preview']['images']:
-            if(len(image_links)<no_of_images):
-                image_links.append(image['source']['url'])
+
+    if 'unsplash' in url:       #For Unsplash
+        soup = make_soup(url)
+        '''Selects desired bs4 tags, soup.select is a recursive function,
+           it searches for classes/tags within classes/tags'''
+        a_tags = soup.select('.y5w1y .hduMF .tPMQE a')
```

Comment:
does unsplash have an API? i guess they do right? i think we can merge this for now and open an issue to use that later, im saying that because they can change there css at any time, which would make, this obsolete :P